app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if @review.errors.present? %>
+<div data-alert class="large-12 alert-box alert" data-alert>
+  <ul>
+    <% @review.errors.full_messages.each do |error| %>
+    <li> <%= error %> </li>
+    <%- end -%>
+  </ul>
+<a href="#" class="close">&times;</a>
+</div>
+<%- end -%>
```

Comment:
Instead of needing to put this at the top of each page where you might be flashing a message, remove this, add `flash[:alert] = @review.errors.full_messages` to your controller, and add the code you deleted back into `application.html.erb`